pkg/controller/nodes/task/handler_test.go

```diff
@@ -806,8 +815,17 @@ func Test_task_Handle_Catalog(t *testing.T) {
 			}
 			tk, err := New(context.TODO(), mocks.NewFakeKubeClient(), c, promutils.NewTestScope())
 			assert.NoError(t, err)
+			fakePlugin := fakeplugins.NewPhaseBasedPlugin()
 			tk.plugins = map[pluginCore.TaskType]pluginCore.Plugin{
-				"test": fakeplugins.NewPhaseBasedPlugin(),
+				"test": fakePlugin,
+			}
+			tk.taskMetricsMap = map[pluginCore.TaskType]*taskMetrics{
+				"test_" + fakePlugin.GetID(): &taskMetrics{
+					taskSucceeded: labeled.NewCounter("success",
+						"Task finished successfully", promutils.NewTestScope(), labeled.EmitUnlabeledMetric),
```

Comment:
Hey I do not see my comment. Can we reword the description to have the tasktype information.